<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="NavWidget" name="map" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>579</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>620</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>出发地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sourceBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>目的地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="destBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>策略</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stg">
        <item>
         <property name="text">
          <string>最短距离</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>途径最短距离</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>最短时间</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交通工具最短时间</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="TimeController" name="timeControl" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>173</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="startTime"/>
    </item>
    <item>
     <widget class="QLineEdit" name="pathTime"/>
    </item>
    <item>
     <widget class="QLineEdit" name="endTime"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="selected">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>171</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavWidget</class>
   <extends>QWidget</extends>
   <header location="global">navwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeController</class>
   <extends>QWidget</extends>
   <header location="global">timecontroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
